rln-cli/src/state.rs

```diff
@@ -1,6 +1,23 @@
+use color_eyre::Result;
 use rln::public::RLN;
+use std::fs::File;
+
+use crate::config::{InnerConfig, Config};
 
 #[derive(Default)]
 pub(crate) struct State<'a> {
     pub rln: Option<RLN<'a>>,
 }
+
+impl<'a> State<'a> {
+    pub(crate) fn load_state() -> Result<State<'a>> {
+        let config = Config::load_config()?;
+        let rln = if let Some(InnerConfig { file, tree_height }) = config.inner {
```

Comment:
are we also storing the tree_config here?

Reply:
Needed to initialise the `RLN` object